bindings/nodejs/src/wallet.rs

```diff
@@ -101,6 +101,7 @@ pub fn call_wallet_method(mut cx: FunctionContext) -> JsResult<JsUndefined> {
     let method_handler = Arc::clone(&&cx.argument::<JsBox<WalletMethodHandlerWrapper>>(1)?.0);
     let callback = cx.argument::<JsFunction>(2)?.root(&mut cx);
 
+    let (sender, receiver) = std::sync::mpsc::channel();
```

Comment:
sync channel in async context? is this on purpose? wouldn't it be better to use an async one?

Reply:
If this is only to set a flag, I would propose using atomic bool, which is also good for async

Reply:
But how do you check the flag then?

Reply:
ok I looked closer, atomic bool isn't solution here, becasue we need to wait for execution of async block

Reply:
The point of this is to get something out of the async block, because neon cx (CallContext) is not Send and therefore can't be used inside of the async block